transport/graphsync/graphsync.go

```diff
@@ -238,6 +238,11 @@ func (t *Transport) consumeResponses(req *gsReq) error {
 	log.Infof("channel %s: finished consuming graphsync response channel", req.channelID)
 
 	for err := range req.errChan {
+		var remoteMissingBlockErr graphsync.RemoteMissingBlockErr
+		if errors.As(err, &remoteMissingBlockErr) {
+			t.events.OnLinkMissing(req.channelID, remoteMissingBlockErr.Link)
+			continue
+		}
```

Comment:
Suggest adding some comments here - I get what it's doing from the explanation in the PR, but someone else reading this code later may not make the connection